### DISCUSSION TITLE
I’m not able to edit my Profile Header Text at this time.

### DISCUSSION THREAD

**Discussion by donwlarson** [community/community#66004] [Other Feature Feedback, Questions, & Ideas]
![image](https://github.com/community/community/assets/1382832/5fd03475-f1e1-4f42-857c-c43f473bfc72)

**Comment by PmnAngelov** [comment]
Have you tried doing it via the desktop version of GitHub? That should work.

**Reply by donwlarson** [reply]
Thank you. That suggestion worked!

Don
